keras_cv/layers/preprocessing/random_crop_and_resize_test.py

```diff
@@ -183,3 +186,91 @@ def test_augment_one_hot_segmentation_mask(self):
         )
         output = layer(inputs, training=True)
         self.assertAllClose(output["segmentation_masks"], input_mask_resized)
+
+    def test_augment_bbox(self):
+        image = tf.zeros([20, 20, 3])
+        bboxes = tf.convert_to_tensor([[[0, 0, 10, 10], [1, 1, 4, 4], [4, 4, 5, 5]]])
+
+        bboxes = bounding_box.add_class_id(bboxes)
+        input = {"images": [image], "bounding_boxes": bboxes}
+        mock_random = [0.1, 0.1, 0.1, 0.1]
+        layer = preprocessing.RandomCropAndResize(
+            target_size=(10, 10),
+            crop_area_factor=(0.5**2, 0.5**2),
+            aspect_ratio_factor=(1.0, 1.0),
+            bounding_box_format="xyxy",
+        )
+
+        with unittest.mock.patch.object(
+            layer._random_generator,
+            "random_uniform",
+            side_effect=mock_random,
+        ):
+            output = layer(input, training=True)
+
+        expected_output = np.asarray(
+            [[[0, 0, 10, 10, 0], [0, 0, 6, 6, 0], [6, 6, 8, 8, 0]]]
+        )
+        expected_output = np.reshape(expected_output, (1, 3, 5))
+        self.assertAllClose(expected_output, output["bounding_boxes"])
+
+    def test_augment_bbox_batched_input(self):
+        image = tf.zeros([20, 20, 3])
+
+        bboxes = tf.convert_to_tensor(
+            [[[0, 0, 10, 10], [4, 4, 12, 12]], [[4, 4, 12, 12], [0, 0, 10, 10]]]
+        )
+        bboxes = bounding_box.add_class_id(bboxes)
+        input = {"images": [image, image], "bounding_boxes": bboxes}
+        mock_random = [0.0, 0.0, 0.1, 0.1]
+        layer = preprocessing.RandomCropAndResize(
+            target_size=(18, 18),
+            crop_area_factor=(0.5**2, 0.5**2),
+            aspect_ratio_factor=(1.0, 1.0),
+            bounding_box_format="xyxy",
+        )
+        with unittest.mock.patch.object(
+            layer._random_generator,
+            "random_uniform",
+            side_effect=mock_random,
+        ):
+            output = layer(input, training=True)
+
+        expected_output = np.asarray(
+            [
+                [[0, 0, 18, 18, 0], [8, 8, 18, 18, 0]],
+                [[8, 8, 18, 18, 0], [0, 0, 18, 18, 0]],
+            ]
+        )
+        expected_output = np.reshape(expected_output, (2, 2, 5))
+        self.assertAllClose(expected_output, output["bounding_boxes"])
+
+    def test_augment_bbox_ragged(self):
+        image = tf.zeros([2, 20, 20, 3])
+        bboxes = tf.ragged.constant(
+            [[[0, 0, 10, 10], [4, 4, 12, 12]], [[0, 0, 10, 10]]], dtype=tf.float32
+        )
+        bboxes = bounding_box.add_class_id(bboxes)
+        input = {"images": image, "bounding_boxes": bboxes}
+        mock_random = [0.0, 0.0, 0.1, 0.1]
+        layer = preprocessing.RandomCropAndResize(
+            target_size=(18, 18),
+            crop_area_factor=(0.5**2, 0.5**2),
+            aspect_ratio_factor=(1.0, 1.0),
+            bounding_box_format="xyxy",
```

Comment:
Do we have a preferred standard on the `bounding_box_format` in tests?

Reply:
thats a good question!  I think my preference would be to try to force a format conversion in unit tests, but no strong opinion.